index.html

```diff
@@ -196,6 +196,39 @@ <h3 class="text1">Language</h3>
       </div>
     </div>
  </section>
+ <footer class="footer">
+<p class="subject">
+    I'm always<br> interested in hearing<br> about new projects,<br> so if you'd like to<br> chat please get in
+</p>
+<form action="https://formspree.io/f/mwkzpjrj" method="post" class="form">
+<div class="full-name">
+    <input type="text" name="name" id="name" maxlength="30" required placeholder="Full name">
+</div>
+<hr class="hor">
+<div class="email">
+    <input type="email" name="email" id="email" required placeholder="Email address">
+</div>
+<hr class="hor">
+<div class="textarea">
+    <textarea name="message" id="message" cols="45" rows="6" required maxlength="500">Enter text here</textarea>
+</div>
```

Comment:
- [ ] Great job the text area has no border on the design, kindly fix that.

### Your Website
![Screen Shot 2022-08-31 at 12 05 35 PM](https://user-images.githubusercontent.com/59416136/187642206-8bf65ec4-f397-4738-9c72-19e92301cb75.png)

### Fimga's Design
![Screen Shot 2022-08-31 at 12 08 24 PM](https://user-images.githubusercontent.com/59416136/187642445-597c658c-2a70-4719-9cb6-d228ef4ccb75.png)